.clang-format

```diff
@@ -4,17 +4,106 @@ IndentWidth: 4
 ---
 Language: Cpp
 BasedOnStyle: LLVM
-IndentWidth: 4
 AccessModifierOffset: 0
```

Comment:
Suggest: `-4`

Reply:
For clarity, this will make `public` etc. look like this:
```cpp
class Foo
{
public:
    Foo();
private:
    void f();
};
```

Reply:
Note: This was about `AccessModifierOffset`.